calculator1.rb

```diff
@@ -0,0 +1,79 @@
+# ******************
+# Stage 1 Calculator
+# ******************
+
+# Method for calculator
+def calculator(operation, num1, num2)
```

Comment:
This method could be broken up into 6 methods.  It's trying to do too many things.